sim/druid/swipe.go

```diff
@@ -102,17 +101,11 @@ func (druid *Druid) newSwipeBearSpellConfig(swipeRank SwipeRankInfo) core.SpellC
 		ThreatMultiplier:             SwipeThreatMultiplier,
 
 		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
+			damage := baseDamage + .1*druid.GetStat(stats.AttackPower)
```

Comment:
Usually for this you want to use `spell.MeleeAttackPower()`
```suggestion
			damage := baseDamage + .1*spell.MeleeAttackPower()
```